content/en/docs/distributions/nutanix/install-kubeflow.md

```diff
@@ -16,6 +16,34 @@ weight = 4
 
 * Download [Kubeconfig](https://portal.nutanix.com/page/documents/details?targetId=Karbon-v2_2:kar-karbon-download-kubeconfig-t.html) of your deployed Karbon cluster. 
 
+* (Optional) Setup [MetalLB](https://metallb.universe.tf/) load balancer for your deployed Karbon cluster.
```

Comment:
Can we move this after the kubeflow is installed but before accessing central dashboard section?